File: detect-pangram/detectPangram.js
const detectPangram = (string) => {
    const alpha = "abcdefghijklmnopqrstuvwxyz".split('');

    for(let i = 0; i < alpha.length; i++){
        if(string.toLowerCase().indexOf(alpha[i]) < 0) return false;
    };
    return true;
};

module.exports = { detectPangram };

//https://www.codewars.com/kata/545cedaa9943f7fe7b000048/train/javascript

File: python-venv/detect_pangram/kata_function/detect_pangram.py
def detect_pangram(string):
    alpha = "a.b.c.d.e.f.g.h.i.j.k.l.m.n.o.p.q.r.s.t.u.v.w.x.y.z".split('.')

    for i in range(len(alpha)):
        if alpha[i] not in string.lower():
            return False

    return True

#https://www.codewars.com/kata/545cedaa9943f7fe7b000048/train/python